packs/bitbucket/README.md

```diff
@@ -175,3 +175,29 @@ Usage:
 ```bash
 st2 run bitbucket.list_branches repo="<repo_name>"
 ```
+
+## Rules
+
+### Post-Receive WebHook
```

Comment:
Can you please also document that this rule needs to be explicitly enabled (since it's disabled by default)?

Besides that, LGTM